src/scss/features/borders.scss

```diff
@@ -2,3 +2,6 @@
 	--divider-width:0px;
 	--tab-outline-width: 0px;
 }
+body:is(.borders-none) .mod-root .workspace-tab-header-container:is(div, :hover){

```

Comment:
regarding the "body:is(.borders-none)",
it's beyond me why replacing it with .borders-none doesn't work.